src/components/ModelGroupToggle/ModelGroupToggle.test.js

```diff
@@ -1,23 +1,34 @@
 import React from "react";
-import { shallow } from "enzyme";
+import { mount } from "enzyme";
+import { MemoryRouter } from "react-router-dom";
 
 import ModelGroupToggle from "./ModelGroupToggle";
 
 describe("Model group toggle", () => {
   it("renders without crashing and matches snapshot", () => {
-    const wrapper = shallow(<ModelGroupToggle />);
+    const wrapper = mount(
```

Comment:
These updated tests create unusable snapshot updates....

https://github.com/canonical-web-and-design/jaas-dashboard/issues/277